Projet/render/ObjetMesh.cpp

```diff
@@ -1,7 +1,7 @@
 #include "StdAfx.h"
```

Comment:
line is irrelevent but why do we still have class ObjetMesh ?

Reply:
They gone!